shell/common/animator_unittests.cc

```diff
@@ -73,6 +73,9 @@ TEST_F(ShellTest, VSyncTargetTime) {
   fml::TaskRunner::RunNowOrPostTask(shell->GetTaskRunners().GetUITaskRunner(),
                                     [engine = shell->GetEngine()]() {
                                       if (engine) {
+                                        // Engine needs a surface for frames to
+                                        // be scheduled.
+                                        engine->OnOutputSurfaceCreated();
```

Comment:
This is a slight change in behavior here that broke this test - before, the test never created a surface so it never actually scheduled frames and the call below this was a no-op. Now, there's an implicit call to RequestFrame(false) in this call. @iskakaushik